### PR TITLE
ISSUE-66: Automatically pre-label uploaded corpus

### PR DESCRIPTION
**Pull request by alimuhammadahmer**
This is a pull requests which contains Resql and Ruuter files required to pre-annotate a corpora using the Anonymiser API.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,18 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/regex.ym
...
 ```
+
+### Automatically pre-label uploaded corpus
+
+https://github.com/buerokratt/Data-Anonymizer/issues/66
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/annotate_corpora.yml
+
+```
+```
```
Comment (turnerrainer) [inline README.md:24]: Please add a sample curl request.

File: README.md [hunk 2]
```diff
@@ -13,3 +13,18 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/regex.ym
...
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/get_latest_corpora.sql
+
+```
+```
```
Comment (turnerrainer) [inline README.md:29]: Please add a sample curl request.

**Review by turnerrainer — CHANGES_REQUESTED** [review]

File: DSL.Ruuter/POST/annotate_corpora.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+annotate_corpora:
+  call: http.post
+  args:
+    url: http://web:5001/annotate_corpora
```
Comment (turnerrainer) [inline DSL.Ruuter/POST/annotate_corpora.yml:4]: What endpoint is `http://web:5001/annotate_corpora`?

File: README.md (hunk 2 above)
Comment (alimuhammadahmer) [reply README.md]: Done

File: DSL.Ruuter/POST/annotate_corpora.yml (hunk 3 above)
Comment (alimuhammadahmer) [reply DSL.Ruuter/POST/annotate_corpora.yml:4]: This is the endpoint which is hosted on our Anonymiser Service. It pseudonymises a particular text using the trained model.

**Review by turnerrainer — APPROVED** [review]

**Pull request merged**